Microsoft.Toolkit.Uwp.UI.Controls/InAppNotification/InAppNotification.Events.cs

```diff
@@ -47,9 +54,29 @@ private void OpenAnimationTimer_Tick(object sender, object e)
         {
             _animationTimer.Stop();
             Opened?.Invoke(this, EventArgs.Empty);
+            SetValue(AutomationProperties.NameProperty, StringExtensions.GetLocalized("/Microsoft.Toolkit.Uwp.UI.Controls/Resources", "WindowsCommunityToolkit_InAppNotification_NameProperty"));
+            peer = FrameworkElementAutomationPeer.CreatePeerForElement(ContentTemplateRoot);
```

Comment:
I asking before if we could cache this, I'm trying to see where ContentTemplateRoot comes from.  We can worry about caching later, so I'd put this back in the `AutomateTextNotification` method for now to keep it where it's used.